src/query/graphite/graphite/metric_type.go

```diff
@@ -0,0 +1,117 @@
+package graphite
+
+import (
```

Comment:
Actually wonder if we need the metric type detection at all, think we can possibly do without it.

Let me see where this tricks up from.